Cargo.toml

```diff
@@ -12,3 +12,6 @@ rfd = "0.13"
 libc = "0.2.168"
 serde = { version = "1.0.216", features = ["derive"] }
 serde_json = "1.0.133"
+
```

Comment:
Could you please add a small comment explaining what this option does and include a link to the relevant documentation for better understanding?

Reply:
Sure. Just did it